### DISCUSSION TITLE
No process to report a Security Vulnerability

### DISCUSSION THREAD

**Discussion by 29039** [rustdesk/rustdesk#3104] [General]
I don't have a security vulnerability to actually report, but I noticed that when I was filing an issue for something else through GitHub, it gave the option to  "Report a security vulnerability: Please review our [security policy](https://github.com/rustdesk/rustdesk/security/policy) for more details. -> View policy" but when I actually press on the button it says "Here we should write what to do in case of a security vulnerability"

I would think that for a piece of software which is capable of remote access, it would be best to fil that in on what to do in case someone actually wanted to report a vulnerability!

**Comment by 29039** [comment]
I actually do have a security concern now, how to report?

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/security/policy

**Comment by 29039** [comment]
cheers
